### DISCUSSION TITLE
visual contrast in notice is low making it hard to read

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13475] [Archive]
medium grey text on a light grey background does not stand out, making it difficult to read. Both for older readers and people who have vision problems, it'd be best if a notice had higher contrast between notice background and text colour.

**Comment by azrikahar** [comment]
Perhaps the Info color notice can be a better alternative in the meantime? Also listing all the available colors here in case we'd like to tweak some/all of them:

- Default

   ![chrome_gpWv4A2cle](https://user-images.githubusercontent.com/42867097/169746303-216e8164-d628-4f05-ad58-2412012935df.png)

- Info

   ![chrome_VnYQAZiWAN](https://user-images.githubusercontent.com/42867097/169746309-d09da88f-350f-446d-a389-11f536c3d313.png)

- Success

   ![chrome_teCUKAVFWn](https://user-images.githubusercontent.com/42867097/169746285-09d06a00-7fe0-4791-8f43-01cac6e55ce1.png)

- Warning

   ![chrome_ZH1A3oFbjC](https://user-images.githubusercontent.com/42867097/169746261-54cbbb11-c875-463f-a97c-4cc377c873c3.png)

- Danger

   ![chrome_B4WJaBKazp](https://user-images.githubusercontent.com/42867097/169746179-3f2e487e-9e5d-4db1-9a5e-fdef3ab2520e.png)
   
(Maybe the warning text color seems to be unintentionally not changed?)

**Reply by sxsws** [reply]
Thanks for the screenshots, great for comments!

My thoughts:
- there's no reason why the text can't be black for all of the statuses. Maybe the distinguisher between the levels could be the icon colour and the side border
- the default text is too light.
- green on grey is difficult to read, as is the red on grey.
- maybe change the danger and warning icon to a red and amber triangle, and success with a green check/tick?

**Reply by benhaynes** [reply]
I agree. Let's use the border/icon color to identify, and keep the text consistently `--foreground-normal`. I also agree that if an icon is not manually set, that the warning and danger notices should use the warning-triangle-outline icon.

For "default", let's make the background color the same as the others and add a gray border (darker than the background). I'm not sure about changing the text color here, I agree it should be darker, but I like the very subdued as it stands out less.

**Reply by sxsws** [reply]
> For "default"... I'm not sure about changing the text color here, I agree it should be darker, but I like the very subdued as it stands out less.

Visually the subdued / muted effect has a lot to answer for (as does bootstrap!) - could there be a `muted` option in the field setup, and a class is added in the vue template if checked to make the text lighter, but make it black by default?

**Reply by benhaynes** [reply]
Yeah, I think `info` could be the default, and we have a `subtle` or `muted` option for the other.
